KhlebnikovAA/Lab_2/Lab_2.cpp

```diff
@@ -0,0 +1,60 @@
+﻿#include <iostream>
+#include <ctime>
+void my_sort(int* arr, int size ) {
+    int i=0, j;
+    do {
+        j = 0;
+        do {
+            if (arr[j+1] < arr[j]) {
+                arr[j + 1] = arr[j] + arr[j + 1];
```

Comment:
That kind of "swap" works only for numbers. Please use more generic approach with temp variable